pkg/controller/servicebindingrequest/nested/path.go

```diff
@@ -0,0 +1,110 @@
+package nested
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Field is the representation of a node in a path.
+type Field struct {
+	// Name is the field name.
```

Comment:
how does it relate to the node?

Reply:
Which node are you referring to? Perhaps node is the wrong term since the data-structure is not a graph nor a tree, but it was meant to indicate the parts (as in "a", "b" and "c" in "a.b.c").